.gitignore

```diff
@@ -44,3 +44,6 @@ Config.toml
 
 # Ignore Gradle build output directory
 build
+
+# Environment files
+*.env
```

Comment:
```suggestion
*.env

```

Reply:
Addressed in [d83f05f](https://github.com/ballerina-platform/module-ballerinax-azure-service-bus/pull/205/commits/d83f05f2a180df4e2fa1dd36ee1b277ff7cda061)